### ISSUE TITLE
Tree Key

### ISSUE THREAD

**Issue by noneck** [louh/nyc-trees#1]
On the map, can you put a key to the tree types or allow the user to click on the tree and read about it?

**Comment by louh** [comment]
Yeah, I'd like to put click-the-tree-for-info in. It will take some work though. I can also link to the Tangram Play version of this which has that kind of interactivity built in: 
https://mapzen.com/tangram/play/?scene=https%3A%2F%2Fcdn.rawgit.com%2Flouh%2Fnyc-trees%2Fb00493901878d130baf7c3b40fc83eccdd523f62%2Fassets%2Ftrees.yaml#18.31666666666669/40.73704/-73.98515
